src/footer/css/footer.pcss

```diff
@@ -80,6 +83,88 @@
 /* Global Footer: Sections */
 /* ========================================================================== */
 
+.esri-footer-section--0 {
+
+	&.hidden{
+		display: none;
+	}
+
+	@media (--gfoot-s) {
+		
+	}
+
+	@media (--gfoot-m) {
+		
+	}
+
+	@media (--gfoot-l) {
+		padding-left: calc(50% - 1424px / 2);
+		padding-right: calc(50% - 1424px / 2);
+	}
+
+}
+
+.esri-footer-breadcrumb {
+	box-sizing: border-box;
+	padding: 25px;
+}
+
+.clear {
+	clear: both;
+}
+
+.esri-footer-breadcrumb--list {
+	float: left;
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	width: 90%;
+}
+
+.esri-footer-breadcrumb--pin {
+	box-sizing: border-box;
+	color: #6e6e6e;
+	float: left;
+	max-width: 15px;
+	padding-right: 25px;
+	width: 5%;
+}
+
+.esri-footer-breadcrumb--pin:hover .svg-icon {
```

Comment:
we should use a different class for the svg-icon, I think there will be collisions there with Calcite Web